.jenkins/cscs/env-icc.sh

```diff
@@ -19,6 +20,7 @@ export CXX=`which CC`
 export CC=`which cc`
 
 configure_extra_options="-DCMAKE_BUILD_TYPE=Debug"
+configure_extra_options+=" -DHPX_WITH_CXX${CXX_STD}=ON"
```

Comment:
Leave as default? I don't mind specifying it explicitly, but we should leave at least one builder that uses the default. We had trouble in the past with the default and explicitly specifying the standard behaving differently.

Reply:
Agreed